teos/teosd.py

```diff
@@ -437,10 +441,10 @@ def main(config):
             if opt in ["--datadir"]:
                 data_dir = os.path.expanduser(arg)
             if opt in ["--wsgi"]:
-                if arg in ["gunicorn", "flask"]:
+                if arg in ["gunicorn", "waitress"]:
                     command_line_conf["WSGI"] = arg
                 else:
-                    exit("wsgi must be either gunicorn or flask")
```

Comment:
Commenting here in lack of a better place: L19 in /teos/help.py and the comment in L293 need to be updated to talk about `waitress` instead of `flask`.